pkgs/development/compilers/ligo/default.nix

```diff
@@ -24,6 +24,8 @@ ocamlPackages.buildDunePackage rec {
     fetchSubmodules = true;
   };
 
+  patches = [ ./linol.patch ];
```

Comment:
Please add a comment or name the patch appropiately to indicate why the patch is needed, or what it does.

Reply:
Thanks for your careful review. I think your comments have been addressed.